### ISSUE TITLE
water

### ISSUE THREAD

**Issue by Lili-Cao** [gnina/gnina#262]
### Issue summary
Hi,

I am wondering if water molecules in a protein pdb file would be sampled during gnina docking process, with refinement used only in the rescore level and not whole protein docking.  Will coordinates of water and non water residues in the protein would change or not, during the docking process?

My second question is , what is the difference between whole protein docking and not whole protein docking?
is the exhaustiveness options determining whether its a whole protein docking or not?

https://github.com/gnina/gnina
gnina -r rec.pdb -l lig.sdf --autobox_ligand rec.pdb -o whole_docked.sdf.gz --exhaustiveness 64

Thanks!
Lili
### Your system configuration
Operating system:
Compiler:
CUDA version (if applicable):
CUDNN version (if applicable):
Python version:

**Comment by dkoes** [comment]
No, everything in the receptor is treated as rigid and is not sampled, unless you specifically enable flexibility, but that will only sample internal degrees of freedom and will not work on a water model.

Whole protein docking is when you define the binding box to be the whole protein as you are doing instead of a specific binding site.  As this is a much larger search space you should increase the exhaustiveness.

**Comment by Lili-Cao** [comment]
Hi

Thanks!
I see the difference in the command line now : )
How does the Gnina treat water molecule then, as well as ions in the protein?
Which forcefield is used to describe the system?  

Thanks!
LC

**Comment by dkoes** [comment]
It treats them as part of the receptor.  By default, the AutoDock Vina scoring function is used for search and refinement and the CNN scoring function for ranking and scoring.

**Comment by Lili-Cao** [comment]
cool.
Thanks!
